<launch>
 <arg name="device_name"             default="/dev/ttyUSB0"/>
 <arg name="baud_rate"               default="57600"/>

 <arg name="drive_wheel"             default="1"/>
 <arg name="steer_wheel"             default="2"/>

 <arg name="profile_velocity"        default="200"/>
 <arg name="profile_acceleration"    default="50"/>

 <param name="device_name"           value="$(arg device_name)"/>
 <param name="baud_rate"             value="$(arg baud_rate)"/>

 <param name="drive_wheel"            value="$(arg drive_wheel)"/>
 <param name="steer_wheel"           value="$(arg steer_wheel)"/>

 <param name="profile_velocity"      value="$(arg profile_velocity)"/>
 <param name="profile_acceleration"  value="$(arg profile_acceleration)"/>

 <node name="swerve_drive_control" pkg="swerve_drive" type="swerve_drive_control" required="true" output="screen"/>
 </launch>
